third_party_libraries/kml.js

```diff
@@ -440,7 +451,7 @@ L.Util.extend(L.KML, {
 			for (i = 0; i < el.length; i++) {
 				switch(tag) {
 					case 'LineString':
-  						var layer = this.parseLineString(el[i], xml, options);
+						var l = this.parseLineString(el[i], xml, options);
```

Comment:
meh alright, I guess it makes sense to keep from confusing with the later "layer" variable